x-pack/plugins/enterprise_search/public/applications/workplace_search/components/shared/credential_item/credential_item.tsx

```diff
@@ -19,6 +19,21 @@ import {
   EuiFieldPassword,
   EuiToolTip,
 } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
```

Comment:
I know App Search does things differently, and I'm always up for re-evaluating how we do things, but for consistency's sake, can we move these to a contestants file?

Reply:
Replied in a different thread: https://github.com/elastic/kibana/pull/112136#discussion_r709145562